### ISSUE TITLE
TSB web scraping sometimes failing with 403 status

### ISSUE THREAD

**Issue by 1jamesthompson1** [1jamesthompson1/TAIC-report-summary#272]
## Problem

The scraping of TSB is done with hrequests. However some of these requests when run on the github action runner (automated testing) are returning 403. 

## Ideas and suggestions

I have looked and this status code can be used when you hnave been blocked by a bot deterctor.

It is intermittent so for now I will leave it as it is.

**Comment by 1jamesthompson1** [comment]
No longer relevant
